sources/packages/web/src/composables/useDesignationAgreement.ts

```diff
@@ -0,0 +1,21 @@
+import { StatusShipTypes } from "@/components/generic/StatusChip.models";
+import { DesignationAgreementStatus } from "@/types/contracts/DesignationAgreementContract";
+
+export function useDesignationAgreement() {
+  const mapDesignationChipStatus = (
+    status: DesignationAgreementStatus,
```

Comment:
mm, I do not understand the reason for this mapping. can we sync up?

Reply:
The idea is that the StatusChipTypes is generic in the way that for different statuses (Designation, Restriction, COE, etc.) green status (StatusChipTypes.success) could be mapped to whatever is needed for the particular Designation, Restriction, COE, etc, status. We can sync up on this.